net/goeap_proxy/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=goeap_proxy
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:= Gabriel Rodriguez <lifehacksback@gmail.com>
+PKG_LICENSE:=none
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/pyther/goeap_proxy.git
+PKG_MIRROR_HASH:=e4014a8dc317d3325d02c1ae81678d6552628bd7a3e253c8ad01a6912ec49dbf
+PKG_SOURCE_VERSION:=3abd66cf35c70b86fe75989ca64c7a72cc8fa797
```

Comment:
Suggest to use codeload, e.g. [rclone](https://github.com/openwrt/packages/blob/3570a61cd330e95d98700c4ae5630406226a8026/net/rclone/Makefile#L12-L14), or check out the tag for `PKG_VERSION`, e.g. [irtt](https://github.com/openwrt/packages/blob/3570a61cd330e95d98700c4ae5630406226a8026/net/irtt/Makefile#L11-L14).